.github/workflows/build-test-deploy.yml

```diff
@@ -450,10 +450,11 @@ jobs:
       - name: Commit changes
         run: |
           git config --local user.email "action@github.com"
-          git config --local user.name "GitHub Action"
+          git config --local user.name "GitHub Action"        
+          git pull      
           git add publiccode.yaml
           git commit -m "${{ github.workflow }}" || echo "No changes to commit"
-          git push    
+          git push --force    
```

Comment:
Hmm, I guess this will fail since we do not allow force pushes on our main branch due to our GitHub ruleset on the main branch.. We should either make an exception for that somehow (not sure if that's even possible) or don't force push and hope it will succeed most of the time.. I think when two main builds are nearly at the same time we can still get conflicts at some point. Not a big issue really if this publiccode.yaml is not 100% up to date at all times..

Reply:
Ah no. Not true. We already by-pass the GitHub main branch ruleset here so a force push should be fine.

```
     # check out the repository with the private SSH key of the deploy key so that
      # we can bypass the ruleset for the main branch
      # see: https://github.com/sbellone/release-workflow-example for details
```